index.js

```diff
@@ -80,6 +82,48 @@ Bonne journée,
     applyLink,
     makeMailto,
   })
+})
+
+app.get("/mission-sante/:id", (req, res) => {
```

Comment:
```suggestion
app.get("/missions/:id", (req, res) => {
```
On peut faire prévoir la généralisation tout de suite.